### ISSUE TITLE
Scrollbar theming

### ISSUE THREAD

**Issue by jil24** [ThomasFKJorna/zotero-night#40]
The add-in works amazingly. Thank you. Is it possible to do anything about the scrollbar color? I am on Mac OS, but looking at screenshots on other OSs the scrollbars also don't appear dark.

zotero-night scrollbars:
![image](https://user-images.githubusercontent.com/11326697/171143992-f7041ddc-8daa-4fec-8aa6-38977c9614f8.png)

firefox scrollbars, for comparison:
![image](https://user-images.githubusercontent.com/11326697/171144279-864b732a-2df6-4912-ab08-a6e982b5033b.png)

**Comment by ThomasFKJorna** [comment]
Yeah this is a good q, I actually don't know why the searchbars aren't dark. I resorted to setting scrollbar visibilty to "scroll only". Maybe I need to set the background of the main document dark?

It might be that this is something fucky with the type of "web page" that Zotero is using, which has been an endless source of frustration. It doesn't follow the normal HTML5 spec but uses some custom version of Mozilla's XUL, which means it doesn't have stuff like `document.body`, which MacOS might use to determine the background color. I'll have a looksy

**Comment by jil24** [comment]
I wonder if it has anything to do with this wontfix? 
https://bugzilla.mozilla.org/show_bug.cgi?id=1513602

**Comment by tefkah** [comment]
I will fix this when version 7 comes out, as newer Firefox allows you to theme scrollbars. This means that I won't fix this in version 6!

https://www.zotero.org/support/dev/zotero_7_for_developers
